expression/aggregation/agg_to_pb.go

```diff
@@ -69,42 +69,45 @@ func AggFuncToPBExpr(sctx sessionctx.Context, client kv.Client, aggFunc *AggFunc
 		tp = tipb.ExprType_StddevSamp
 	}
 	if !client.IsRequestTypeSupported(kv.ReqTypeSelect, int64(tp)) {
-		return nil
+		return nil, errors.New("select request is not supported by client")
 	}
 
 	children := make([]*tipb.Expr, 0, len(aggFunc.Args))
 	for _, arg := range aggFunc.Args {
 		pbArg := pc.ExprToPB(arg)
 		if pbArg == nil {
-			return nil
+			return nil, errors.New(aggFunc.String() + " can't be converted to PB.")
 		}
 		children = append(children, pbArg)
 	}
+	retType, err := expression.ToPBFieldTypeWithCheck(aggFunc.RetTp, storeType)
```

Comment:
Why not just add the return type check to `aggregation.CheckAggPushDown`?

Reply:
// CheckAggPushDown checks whether an agg function can be pushed to storage.
from the comments, we known it regardless the datatype, just check the function name.

Reply:
Oh, my mistake, what I want to say is why not just add the return type check to `aggregation.CheckAggCanPushCop`

Reply:
there are total 3 places call `aggregation.AggFuncToPBExpr`, including `core.CheckAggCanPushCop`. It is better to checke the return type in `AggFuncToPBExpr`.